src/app/redux/actions/cellActions.js

```diff
@@ -171,10 +171,65 @@ const dispatchCellValueChange = action => (dispatch, getState) => {
       });
     }
   })
+    .then(() =>
+      maybeUpdateStatusColumnValue(tableId, columnId, rowId)(dispatch, store)
+    )
     .then(() => refreshDependentRows(tableId, [rowId], store.getState()))
     .then(state => dispatch({ type: SET_STATE, state }));
 };
 
+const maybeUpdateStatusColumnValue = (tableId, columnId, rowId) => (
+  dispatch,
+  store
+) => {
+  const state = store.getState();
+  const calcDependentColumnIds = conditions => {
+    return f.flatMap(condition => {
+      return f.has("column", condition)
+        ? condition.column
+        : calcDependentColumnIds(condition);
+    }, conditions.values);
+  };
+  const statusColumns = f.filter(
+    column => column.kind === ColumnKinds.status,
+    state.columns[tableId].data
+  );
+  if (f.isEmpty(statusColumns)) {
+    return;
+  }
+  return f.compose(
+    f.forEach(({ column, dependentColumnIds }) => {
```

Comment:
[suggestion] Hier könntest du, statt nur die Effekte mit `forEach` rauszuhauen, die Promises mappen und noch einen Schritt `ps => Promise.all(ps)` dahinterhängen.
Prinzipiell sind Promises in Actions irgendwie chainbar (auch wenn das fast nie verwendet wird und ich mir über die Implementierung nicht mehr im Klaren bin), weil sie als Thunks gespeichert bleiben, so würde das Pattern nicht brechen.